### DISCUSSION TITLE
xampp and symfony 6.1

### DISCUSSION THREAD

**Discussion by MoniqueHahnefeld** [symfony/symfony#47759] [Q&A]
Hi, I have a problem with this warning:
"The given 'wrapperClass' App\Entity\DatabaseConnection has to be a subtype of \Doctrine\DBAL\Connection."
This class App\Entity\DatabaseConnection does not longer exist, but I get this Error all the time. Tryed everything. I don't know if it is an xampp cache issue or what happens there. Its weird. Do somebody know such issue and can maybe help me?
Thx

**Comment by ahundiak** [comment]
Do you have a wrapper_class specified in config/packages/doctrine.yaml?

**Reply by MoniqueHahnefeld** [reply]
Thank you. That I forgot :-)
